tidb-configuration-file.md

```diff
@@ -199,6 +205,7 @@ Configuration items related to log.
 - Determines whether to enable the slow query log.
 - Default value: `true`
 - To enable the slow query log, set `enable-slow-log` to `true`. Otherwise, set it to `false`.
+- Since v6.1.0, whether to enable slow query log is determined by the TiDB configuration item [`instance.tidb_enable_slow_log`](/tidb-configuration-file.md#tidb_enable_slow_log) or the system variable [`tidb_enable_slow_log`](/system-variables.md#tidb_enable_slow_log). `enable-slow-log` still takes effect. But if `enable-slow-log` and `instance.tidb_enable_slow_log` are set at the same time, TiDB will apply the value of `instance.tidb_enable_slow_log`.
```

Comment:
```suggestion
- Since v6.1.0, whether to enable slow query log is determined by the TiDB configuration item [`instance.tidb_enable_slow_log`](/tidb-configuration-file.md#tidb_enable_slow_log) or the system variable [`tidb_enable_slow_log`](/system-variables.md#tidb_enable_slow_log). `enable-slow-log` still takes effect. But if `enable-slow-log` and `instance.tidb_enable_slow_log` are set at the same time, the latter takes effect.
```